charts/k8ssandra/values.yaml

```diff
@@ -156,30 +104,87 @@ cass-operator:
   enabled: true
 
 kube-prometheus-stack:
-  alertmanager:
+  # Disable default service monitors
+  coreDns:
     enabled: false
-
-  kubeStateMetrics:
+  kubeApiServer:
     enabled: false
-
-  grafana:
+  kubeControllerManager:
     enabled: false
-
-  nodeExporter:
+  kubeDns:
     enabled: false
-
-  kubeletService:
+  kubeEtcd:
     enabled: false
-
-  kubeControllerManager:
+  kubeProxy:
+    enabled: false
+  kubeScheduler:
+    enabled: false
+  kubeStateMetrics:
     enabled: false
-
   kubelet:
     enabled: false
+  nodeExporter:
+    enabled: false
 
-  kubeApiServer:
+  alertmanager:
     enabled: false
+    serviceMonitor:
+      selfMonitor: false
+
+  # Locks Prometheus operator to this namespace and kube-system
+  prometheusOperator:
```

Comment:
Don't we need the `prometheusOperator.enabled` flag exposed?

Reply:
It defaults to true so I didn't include it here. Users may still override this parameter in their local values.yaml if they want.